### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by maelle** [ropensci/redland-bindings#67]
Hi there 👋 

I was wondering whether the path to the R package could/should be made clearer. Are you interested in a PR with a README.md? 

Unless the repo can become a separate R package repo of course 😉

**Comment by gothub** [comment]
Hi @maelle - yes, a ./README.md is a great idea. Also, the CRAN entry for the `redland` package doesn't list a URL indicating where the source is, so I can update that with the next CRAN release.

BTW, the development environment instructions in ./R/README.md could use a refresh to (on my list now).

In regards to ./R/redland being split into another repo, maybe @cboettig and @mbjones could comment on that.f

**Comment by mbjones** [comment]
I would prefer to keep it where it is because that is the approach used by
Redland bindings with other languages. Plus we still have upstream patches
that haven't landed.

Matt

**Comment by maelle** [comment]
Thanks both! 😺 

@gothub will you have time to write ./README.md? Otherwise I can try writing it by looking at the docs.

**Comment by mbjones** [comment]
There is already a Readme.md in the R/redland directory. The top level directory is set by the redland-bulindings project. Is that where you want the readme?

**Comment by maelle** [comment]
I was thinking of a Markdown README on the landing page of the repo instead of the README.html that's not so nice to read on GitHub.

**Comment by cboettig** [comment]
@maelle I don't think we should really change things at the top-level directory as long as this repo is a fork of the whole `redland-bindings`.  

On the other hand, I'm not sure how realistic is that our R bindings addition will ever be merged back into dajobe/redland-bindings anyway given the absence of activity over there.   Maybe it is still worth respecting that package structure anyway out of respect for the original developer's model, but I agree with @maelle that it does make this repo a bit less intuitive to R users.

(Perhaps related to the absence of activity or perhaps irrelevant, but seems that the primarily used RDF implementations in other languages, e.g. python's rdflib or ruby rdf, aren't using these swig-based bindings either? though maybe just because they are (appear to be) stand-alone native-language implementations; but just curious why the redland bindings versions didn't see as much adoption?)

**Comment by maelle** [comment]
Maybe a compromise would be to have a nice landing page for the R package somewhere else and use it as the URL near repo description (i.e. in this fork replacing "Redland librdf language bindings http://librdf.org/bindings/" with "R package for Redland librdf <URL to an R package README>")?

**Comment by gothub** [comment]
I would think that most R users that are interested in viewing the source for the R `redland` package would find the location via the CRAN page: https://cran.r-project.org/web/packages/redland. If the `redland` package description includes this URL, then users will get to the correct page, and we can put any desired explanation of how the R package relates to the redland-bindings and redland C libraries.

How does this sound?

**Comment by maelle** [comment]
It sounds good! 🙂

**Comment by mbjones** [comment]
Sounds good to me too -- did these documentation changes end up in the release you just submitted to CRAN, or are you targeting a future release?
